jgrapht-io/src/main/java/org/jgrapht/io/DOTUtils.java

```diff
@@ -38,7 +38,7 @@
     static final String UNDIRECTED_GRAPH_EDGEOP = "--";
 
     // patterns for IDs
-    private static final Pattern ALPHA_DIG = Pattern.compile("[a-zA-Z]+([\\w_]*)?");
+    private static final Pattern ALPHA_DIG = Pattern.compile("[a-zA-Z_][\\w]*");
```

Comment:
Thanks.  According to

https://www.graphviz.org/doc/info/lang.html

we still won't be in full conformance since they allow a bunch of stuff like letters with diacritical marks as well (\200-\377).  Not sure if anyone cares.

Reply:
Yeah, we'll leave it as is until someone cares.